test/custom-field-basic-test.html

```diff
@@ -44,9 +44,9 @@
       });
 
       it('should properly define internal inputs', function() {
-        expect(customField.__inputs.length).to.equal(2);
+        expect(customField.querySelectorAll('input').length).to.equal(2);
         for (var i = 0; i < 2; i ++) {
-          expect(customField.__inputs[i].localName).to.be.equal('input');
+          expect(customField.querySelectorAll('input')[i].localName).to.be.equal('input');
```

Comment:
note: this test doesn't make sense anymore after the change suggested by me. Wondering how should we publicly detect that the particular input is managed by `vaadin-custom-field` 🤔

Reply:
Yeah, my mistake 😅 
Fixed now